agentMET4FOF/metrological_streams.py

```diff
@@ -16,25 +16,12 @@ def _time():
 
     @staticmethod
     def _time_unc():
-        return time.get_clock_info("time").resolution
+        return .01*time.get_clock_info("time").resolution
```

Comment:
I do not understand why the uncertainty should be resolution times <img src="https://render.githubusercontent.com/render/math?math=10^{-2}">. Should we ask the user to specify this as well and expect zero otherwise? If we feel this is not appropriate, we might initialize the uncertainty as None and warn the user about it, as we proposed above for the labels.

Reply:
The value of 0.01 was chosen randomly. It just results in a nicer looking plot.

I agree that the default value should be set to zero and that the users should specify the uncertainty themselves.

Reply:
Ah yes, now I remember that we talked about the nicer looking plot.